torch/_torch_docs.py

```diff
@@ -4108,6 +4108,44 @@
 
 """)
 
+add_docstr(torch.randint,
+           r"""
+randint(low=0, high, sizes, out=None) -> Tensor
+
+Returns a tensor filled with random integers generated uniformly
+between :attr:`low` (inclusive) and :attr:`high` (exclusive).
+
+The shape of the tensor is defined by the variable argument :attr:`sizes`.
+
+Args:
+    low (int, optional): Lowest (positive) integer to be drawn from the distribution. Default: 0.
```

Comment:
Any reason that `low` and `high` need to be positive? `0` is not positive as well.

Reply:
Yeah so in the develop mode 
> >>> a = torch.randn(5,6)
>>> a.random_(-10,-1)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/naman/PythonPackages/Source-Pytorch/lib/python3.6/site-packages/torch/autograd/variable.py", line 84, in __repr__
    return torch._tensor_str._str(self)
  File "/home/naman/PythonPackages/Source-Pytorch/lib/python3.6/site-packages/torch/_tensor_str.py", line 306, in _str
    strt = _matrix_str(self)
  File "/home/naman/PythonPackages/Source-Pytorch/lib/python3.6/site-packages/torch/_tensor_str.py", line 215, in _matrix_str
    min_sz=5 if not print_full_mat else 0)
  File "/home/naman/PythonPackages/Source-Pytorch/lib/python3.6/site-packages/torch/_tensor_str.py", line 95, in _number_format
    if value != math.ceil(value.item()):
RuntimeError: Overflow when unpacking long

So I thought random_ works only for positive numbers. But  just now checked it's overflow problem and works for torch.LongTensor. 
By the way isn't it random_ implementation issue. In default settings for FloatTensors or even DoubleTensors it would generate this message? 
I'll remove positive but I should add warning of overflow then maybe?

Reply:
@li-roy See print error above.

Reply:
@Naman-ntc Let's remove "positive". The error you saw was purely print format error. The actual method works on nonpositive values.

Reply:
@SsnL removed in next commit, but still randint wont work in FloatTensor or DoubleTensor setting right (for negatives)! Is it alright, because if he needs DoubleTensor randint user'll have to typecase it to LongTensor and then to DoubleTensor which is very 'smooth' imo?

Reply:
That is an related bug and shouldn't affect your PR. I opened an issue at #6338

Reply:
Great Thanks a lot :+1:

Reply:
@SsnL  how big would be a patch for #6338 ? If it's not too big I would like try it over a few days.